firstvoices/backend/management/commands/utils/iterators.py

```diff
@@ -35,19 +35,37 @@ def media_doc_generator(instance, media_type):
 def audio_iterator():
     queryset = Audio.objects.all()
     for instance in queryset:
-        yield media_doc_generator(instance, "audio").to_dict(True)
+        try:
+            yield media_doc_generator(instance, "audio").to_dict(True)
+        except AttributeError:
+            print(
```

Comment:
These should probably be logs instead of printouts. Same with the two functions below.

Reply:
Good idea. The reason these are print statements are because this command is always executed from the shell manually so it never goes to a logger or sentry. I used self.stdout.write in the main method to log the "success messages" but to use that I would have to pass the "self" object through multiple methods to reach this point.
I did a bit of research, and print is almost the same as stdout.write (print converts whatever you give it to a string, and then calls the stdout.write internally) so I left it at print.
My main focus to get this merged is to find the problematic files which are missing attributes.

Reply:
I did some more research, while the main was building to reflect the changes. Seems like logger is the way to go. I will update it and raise another PR to add those changes.
Thanks for the suggestion.